api-examples/function.HH.Lib.Math.max_by.md

```diff
@@ -0,0 +1,9 @@
+```
+$items= vec[tuple('a',9), tuple('b',8), tuple('c', 7)];
+$maxItem= Math\max_by($items, $item==> $item[1]);
+echo "The max item is " . $maxItem[0];
+```
+Ouput
+```
+The max item is a
```

Comment:
This confused me for a minute -- the max item is a what? How about calling them foo, bar and baz rather than a, b and c?

Reply:
Haha, indeed, it is confusing. I changed them as you suggested.